psl-core/src/main/java/org/linqs/psl/application/learning/weight/bayesian/GaussianProcessPrior.java

```diff
@@ -217,53 +263,89 @@ private void setWeights(WeightConfig config) {
 
         int numPerSplit = (int)Math.exp(Math.log(maxConfigs) / numMutableRules);
 
+        // Create config for provided weight configuration in model file
+        WeightConfig initialConfig = new WeightConfig(new float[numMutableRules]);
+        for (int j = 0; j < numMutableRules; j++) {
```

Comment:
Default to `i` for the outermost loop if a more descriptive name is not necessary.